packages/grant-explorer/src/features/api/utils.ts

```diff
@@ -194,59 +212,52 @@ export const payoutTokens = [
   ...GOERLI_TESTNET_TOKENS,
   ...FANTOM_TESTNET_TOKENS,
   ...PGN_TESTNET_TOKENS,
+  ...PGN_MAINNET_TOKENS,
 ];
 
-export const getPayoutTokenOptions = (chainId: string): PayoutToken[] => {
-  switch (chainId) {
-    case ChainId.MAINNET: {
-      return MAINNET_TOKENS;
-    }
-    case ChainId.OPTIMISM_MAINNET_CHAIN_ID: {
-      return OPTIMISM_MAINNET_TOKENS;
-    }
-    case ChainId.FANTOM_MAINNET_CHAIN_ID: {
-      return FANTOM_MAINNET_TOKENS;
-    }
-    case ChainId.FANTOM_TESTNET_CHAIN_ID: {
-      return FANTOM_TESTNET_TOKENS;
-    }
-    case ChainId.PGN_TESTNET_CHAIN_ID: {
-      return PGN_TESTNET_TOKENS;
-    }
-    case ChainId.GOERLI_CHAIN_ID:
-    default: {
-      return GOERLI_TESTNET_TOKENS;
-    }
-  }
+type PayoutTokensMap = Record<ChainId, PayoutToken[]>;
+const payoutTokensMap: PayoutTokensMap = {
+  [ChainId.GOERLI_CHAIN_ID]: GOERLI_TESTNET_TOKENS,
+  [ChainId.MAINNET]: MAINNET_TOKENS,
+  [ChainId.OPTIMISM_MAINNET_CHAIN_ID]: OPTIMISM_MAINNET_TOKENS,
+  [ChainId.FANTOM_MAINNET_CHAIN_ID]: FANTOM_MAINNET_TOKENS,
+  [ChainId.FANTOM_TESTNET_CHAIN_ID]: FANTOM_TESTNET_TOKENS,
+  [ChainId.PGN]: PGN_MAINNET_TOKENS,
+  [ChainId.PGN_TESTNET]: PGN_TESTNET_TOKENS,
+};
+
+export const getPayoutTokenOptions = (chainId: ChainId): PayoutToken[] =>
+  payoutTokensMap[chainId];
+
+const graphQlEndpoints: Record<ChainId, string> = {
+  [ChainId.PGN]: process.env.REACT_APP_SUBGRAPH_PGN_API!,
+  [ChainId.GOERLI_CHAIN_ID]: process.env.REACT_APP_SUBGRAPH_GOERLI_API!,
+  [ChainId.PGN_TESTNET]: process.env.REACT_APP_SUBGRAPH_PGN_TESTNET_API!,
+  [ChainId.MAINNET]: process.env.REACT_APP_SUBGRAPH_MAINNET_API!,
+  [ChainId.OPTIMISM_MAINNET_CHAIN_ID]:
+    process.env.REACT_APP_SUBGRAPH_OPTIMISM_MAINNET_API!,
+  [ChainId.FANTOM_MAINNET_CHAIN_ID]:
+    process.env.REACT_APP_SUBGRAPH_FANTOM_MAINNET_API!,
+  [ChainId.FANTOM_TESTNET_CHAIN_ID]:
+    process.env.REACT_APP_SUBGRAPH_FANTOM_TESTNET_API!,
 };
 
 /**
  * Fetch subgraph network for provided web3 network
  *
- * @param chainId - The chain ID of the blockchain2
+ * @param chainId - The chain ID of the blockchain
  * @returns the subgraph endpoint
  */
-const getGraphQLEndpoint = async (chainId: ChainId) => {
-  switch (chainId) {
-    case ChainId.MAINNET:
-      return `${process.env.REACT_APP_SUBGRAPH_MAINNET_API}`;
-
-    case ChainId.OPTIMISM_MAINNET_CHAIN_ID:
-      return `${process.env.REACT_APP_SUBGRAPH_OPTIMISM_MAINNET_API}`;
-
-    case ChainId.FANTOM_MAINNET_CHAIN_ID:
-      return `${process.env.REACT_APP_SUBGRAPH_FANTOM_MAINNET_API}`;
-
-    case ChainId.FANTOM_TESTNET_CHAIN_ID:
-      return `${process.env.REACT_APP_SUBGRAPH_FANTOM_TESTNET_API}`;
-
-    case ChainId.PGN_TESTNET_CHAIN_ID:
-      return `${process.env.REACT_APP_SUBGRAPH_PGN_TESTNET_API}`;
-
-    case ChainId.GOERLI_CHAIN_ID:
-    default:
-      return `${process.env.REACT_APP_SUBGRAPH_GOERLI_API}`;
-  }
+const getGraphQLEndpoint = (chainId: ChainId) => `${graphQlEndpoints[chainId]}`;
```

Comment:
Are the backticks necessary here?

Reply:
Yes, this is the solution to the fetch testing bug we chatted about yesterday. The best solution is to improve the test, but the backticks preserve existing behaviour.

Reply:
If you feel like pushing one more time (can't blame you if you don't :)) please add a comment so one can see the reason for the apparent redundancy.

Reply:
yep, fixed